Sources/CircuitBreaker/CircuitBreaker.swift

```diff
@@ -78,21 +100,27 @@ public class CircuitBreaker<A, B> {
 
   // MARK: Initializers
 
-  /// Initializes CircuitBreaker instance with asyncronous context command (Advanced usage)
+  /// Initializes a CircuitBreaker instance with an asyncronous context command. (Advanced usage.)
```

Comment:
I'm not sure what is meant by (Advanced usage) here. Is using this initializer advances?

Reply:
I've removed the words "Advanced usage" as I think that makes it less confusing.